ledger/accountdb.go

```diff
@@ -4177,10 +4183,11 @@ func processAllResources(
 	callback func(addr basics.Address, creatableIdx basics.CreatableIndex, resData *resourcesData, encodedResourceData []byte) error,
 ) (pendingRow, error) {
 	var err error
+	var buf []byte
+	var addrid int64
+	var aidx basics.CreatableIndex
```

Comment:
Does taking these out also help?

Reply:
yes, they are passed by address to Scan, I expect the same issue with allocations. I was not able to run escape analysis in vscode to confirm